apps/authentication-portal/src/main/webapp/smsOtp.jsp

```diff
@@ -1,5 +1,5 @@
 <%--
-  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
```

Comment:
Since this is not a new file, should we change the year?

Reply:
Reverted the year and the url in the copyright section.